### DISCUSSION TITLE
Customize glossary with group titles

### DISCUSSION THREAD

**Discussion by Marcelo-Alves7** [joaomlourenco/novathesis#288] [Q&A] [answered]
Hello professor,

I was trying to adapt the different glossaries in the template, and I changed the name, for example from "Chemical Symbols" to "Nomenclature", because it makes more sense to me.
However, it would be useful for me to be able to highlight some nomenclature by groups of families, for example, A for a group of numbers and letters G for a group of Greek letters S for a group of subscripts, similar to the image I generated in another document:

![image](https://user-images.githubusercontent.com/98588105/151994380-1d5621b3-1956-4e83-ac18-8648f7aa6495.png)

Can the teacher help me with this question, how could I create groups name in the "Nomenclature" glossary?

Thank You.

**Comment by mmantao2112** [comment]
Hello Marcelo, could you explain me how you changed the glossary name?

**Reply by joaomlourenco** [reply, marked as answer]
Please see the [wiki FAQ](https://github.com/joaomlourenco/novathesis/wiki/FAQ#2-how-to-rename-the-list-ofsomething-to-list-of-whatever)!

**Reply by joaomlourenco** [reply]
Please use `\ntlangsetup*{…}` to force the redefinition.

**Reply by mmantao2112** [reply]
Hello again prof. João,

It is still not working. I've tried already with acronym, acronyms, listofacronym, listofacronyms and acronympl.

![image](https://github.com/user-attachments/assets/9067668d-0023-4a5e-a185-8a66482c24ca)

**Reply by joaomlourenco** [reply]
I tried it before recommending using
```latex
\ntlangsetup*{en/acronym=Acronyms and Abbreviations}
```
and it worked for me.  

So, either you are using an old/outdated version of the template or Murphy is playing with us.  Which version are you using?

**Reply by mmantao2112** [reply]
\newcommand*{\novathesisversion}{7.1.18}
\newcommand*{\novathesisdate}{2024-01-16}

Is the version I am using.

**Reply by joaomlourenco** [reply]
I confirm it does not work in version 7.1.18, but it is working with the latest version.  Please upgrade to the latest version. It is quite easy…. just **make a backup** and then replace the files `novathesis.cls`, `template.tex` and the folder `NOVAthesisFiles`.  You will need to install the newsgott fonts and it shall work (🤞).

**Reply by mmantao2112** [reply]
Thank you. i downloaded the latest version and it worked.
